src/fs/minmax.hh

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+#pragma once
+
+#include <utility>
+
+namespace seastar::fs {
+
+template<class T>
+std::pair<T, T> minmax(T a, T b) {
+    if (a < b)
```

Comment:
My original comment was to just point out the curly braces, but, in fact, moving this utility into a separate header is an overkill. It seems to be used only in a single .cc file, so let's just define it in there. Also, the name does not really explain what this function does (my original guess was something like `std::clamp`), so that's one more reason to keep it hidden in a .cc file :)

Reply:
It resembles `std::minmax`, which is unusable here because of returning crefs. Maybe naming it `sorted` would be better?